SECURITY.md

```
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x.x   | :white_check_mark: |
+| < 1.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities to security@example.com (Replace with actual email).
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)